### PR TITLE
Include an account command with subcommands

### PR DESCRIPTION
**Pull request by tomasmik**
Last commands and sub-commands to finish up with https://github.com/mysteriumnetwork/node/issues/2837

### CODE REVIEW

File: cmd/commands/account/command.go [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+	}
+
+	resp, err := c.tequilapi.OrderCreate(identity.FromAddress(id.Address), contract.OrderRequest{
+		MystAmount:       amount,
+		PayCurrency:      currency,
+		LightningNetwork: false,
+	})
+
+	if err != nil {
+		clio.Error("Failed to create an top up request, make sure your requested amount is equal or more than 0.0001 BTC")
```
Comment (soffokl) [inline cmd/commands/account/command.go:174]: Hm, why does it fixed to 0.0001 BTC? 
```suggestion
		clio.Error("Failed to create an top up request, make sure your requested amount is equal or more than 0.0001 BTC")
```

**Review by soffokl — APPROVED** [review]

File: cmd/commands/account/command.go (hunk 1 above)
Comment (tomasmik) [reply cmd/commands/account/command.go:174]: That not my problem. Thats coingate doing coingate things. Minimum amount of myst is whatever it is converted to 0.0001 BTC at the time of purchase.

File: cmd/commands/account/command.go [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/identity/registry"
+	"github.com/mysteriumnetwork/node/money"
+	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
+	"github.com/mysteriumnetwork/node/tequilapi/contract"
+
+	"github.com/urfave/cli/v2"
+)
+
+// CommandName is the name of register command.
```
Comment (mdomasevicius) [inline cmd/commands/account/command.go:39]: did you mean? register -> registered

File: cmd/commands/account/util.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+package account
+
+import (
+	"fmt"
+
+	"github.com/mysteriumnetwork/node/cmd/commands/cli/clio"
+	"github.com/mysteriumnetwork/node/tequilapi/contract"
+)
+
+func matchString(needle string, stack []string) bool {
```
Comment (mdomasevicius) [inline cmd/commands/account/util.go:27]: API already suggests working with string based types
`matchString()` -> `match("MYST", currencies)` 
`matchString()` -> `contains("MYST", currencies)`

File: cmd/commands/account/command.go [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+
+	ok, err := c.identityIsUnregistered(id.Address)
+	if err != nil {
+		clio.Error(err.Error())
+		return
+	}
+
+	if ok {
+		clio.Warn("Please initiate your identity registration before trying top up")
+		return
```
Comment (mdomasevicius) [inline cmd/commands/account/command.go:164]: Maybe this should be first guard clause?

File: cmd/commands/account/command.go [hunk 5]
```diff
@@ -0,0 +1,261 @@
...
+	}
+
+	ok, err := c.identityIsUnregistered(id.Address)
+	if err != nil {
+		clio.Error(err.Error())
+		return
+	}
+
+	if ok {
+		clio.Warn("Please initiate your identity registration before trying top up")
```
Comment (mdomasevicius) [inline cmd/commands/account/command.go:163]: Maybe "Please initiate your identity registration before trying top up" -> "Please register identity before trying top up" ?

File: cmd/commands/account/command.go [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+	currency := ctx.String(flagCurrency.Name)
+	if !matchString(currency, currencies) {
+		clio.Warn("Given currency cannot be used")
+		clio.Info("Supported currencies are:", strings.Join(currencies, ", "))
+		return
+	}
+
+	amount := ctx.Float64(flagAmount.Name)
+	if amount <= 0 {
+		clio.Warn("Amount to top up must be given and must be more than 0")
```
Comment (mdomasevicius) [inline cmd/commands/account/command.go:146]: `Amount to top up must be given and must be more than 0` -> `Top up Amount is required and must be greater than 0` ?

File: cmd/commands/account/command.go [hunk 7]
```diff
@@ -0,0 +1,261 @@
...
+				Usage: "Create a new top up for your account",
+				Flags: []cli.Flag{&flagAmount, &flagCurrency},
+				Action: func(ctx *cli.Context) error {
+					cmd.topup(ctx)
+					return nil
+				},
+			},
+			{
+				Name:  "info",
+				Usage: "Display information about your account",
```
Comment (mdomasevicius) [inline cmd/commands/account/command.go:99]: `Display information about your account` -> `Display identity account information` ?

**Review by mdomasevicius — APPROVED** [review]

**Pull request merged**